assets/component-image-with-text.css

```diff
@@ -82,13 +82,92 @@
   align-items: flex-start;
   height: 100%;
   justify-content: center;
+  align-self: center;
   padding: 4rem calc(4rem / var(--font-body-scale)) 5rem;
 }
 
+.image-with-text__content--mobile-right > * {
+  align-self: flex-end;
+  text-align: right;
+}
+
+.image-with-text__content--mobile-center > * {
+  align-self: center;
+  text-align: center;
+}
+
+.image-with-text--overlap .image-with-text__content {
+  transform: translate(0 , -3rem);
+  width: 90%;
+  margin: 0 auto; 
+}
+
 @media screen and (min-width: 750px) {
   .image-with-text__grid--reverse .image-with-text__content {
     margin-left: auto;
   }
+
+  .image-with-text__content--bottom {
+    justify-content: flex-end;
+    align-self: flex-end;
+  }
+
+  .image-with-text__content--top {
+    justify-content: flex-start;
+    align-self: flex-start;
+  }
+
+  .image-with-text__content--desktop-right > * {
+    align-self: flex-end;
+    text-align: right; 
+  }
+
+  .image-with-text__content--desktop-left > * {
+    align-self: flex-start;
+    text-align: left;
+  }
+
+  .image-with-text__content--desktop-center > * {
+    align-self: center;
+    text-align: center;
```

Comment:
It's also used for the subtitle and heading. It doesn't look like it because they are short, but if they are long and you do not use text align, then they don't work as expected. 

<img width="506" alt="Screen Shot 2021-12-03 at 2 40 56 PM" src="https://user-images.githubusercontent.com/26778637/144663141-a4a04295-8c85-441a-9d91-da58822d12c1.png">

<img width="508" alt="Screen Shot 2021-12-03 at 2 41 41 PM" src="https://user-images.githubusercontent.com/26778637/144663214-fd8baedc-879c-4289-b9fc-fe46faf87fd9.png">